my_project/global_scheme.py

```diff
@@ -841,10 +843,12 @@
     "Vis",
 ]
 variables_dropdown = [
-    "DBT",
+    "t_wb",
     "DPT",
+    "DBT",
     "RH",
-    "p_atm",
+    "p_vap",
```

Comment:
I saw that you moved the order of some of the variables. Is there a specific reason for this?

Reply:
Hi Federico, I moved the order of some variables because task 154 requires the options on the PSY tab to be in a specific order. For example, the option after "p_vap" on line 850 needs to be "hr".

Reply:
Great then, I will accept the changes and merge the pull request.